### PR TITLE
Include alias location statement support

### PR DESCRIPTION
**Pull request by RaymondSchnyder**
### Proposed changes
Includes the alias statement for locations.
Addresses: https://github.com/nginxinc/ansible-role-nginx-config/issues/131

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

-   [x] I have read the [CONTRIBUTING](https://github.com/nginxinc/ansible-role-nginx-config/blob/main/CONTRIBUTING.md) document
-   [x] I have added Molecule tests that prove my fix is effective or that my feature works
-   [x] I have checked that any relevant Molecule tests pass after adding my changes
-   [x] I have updated any relevant documentation (`defaults/main/*.yml`, `README.md` and `CHANGELOG.md`)

Somehow, i cannot run molecule verify at the moment.

### CODE REVIEW

File: defaults/main/template.yml [hunk 1]
```diff
@@ -764,6 +764,7 @@ nginx_config_http_template:
               #   value: Value-X
               #   always: false
             root: /usr/share/nginx/html
+            # alias: /usr/share/nginx/html
```
Comment (RaymondSchnyder) [inline defaults/main/template.yml:767]: Is this okay like this? Im not sure if i should uncomment this.

Comment (alessfg) [reply defaults/main/template.yml:767]: I'd leave it uncommented. I usually only comment out variables when there's various configuration options.

File: molecule/default/verify.yml [hunk 2]
```diff
@@ -56,6 +56,15 @@
       register: conf
       failed_when: (conf is changed) or (conf is failed)
 
+    - name: Ensure frontend_default.conf contains 'alias directive'
+      lineinfile:
+        path: /etc/nginx/conf.d/frontend_default.conf
+        line: "    alias /usr/share/nginx/html;"
```
Comment (alessfg) [inline molecule/default/verify.yml:62]: This test is failing. You are probably missing four whitespaces 😄

**Review by alessfg — CHANGES_REQUESTED** [review]

**Comment by alessfg** [comment]
Overall this PR looks solid. Once you make the change required for the test to pass, I'll merge it 😄

**Comment by RaymondSchnyder** [comment]
@alessfg What do i need to do to rerun the Molecule tests here?
They run through on my local installation.

**Review by alessfg — APPROVED** [review]

**Comment by alessfg** [comment]
I had to manually approve the tests to run 👀
Everything looks good so I'm merging it. Thanks for your contribution!

**Pull request merged**